nilearn/_utils/cache_mixin.py

```diff
@@ -213,10 +228,6 @@ def _cache(self, func, func_memory_level=1, shelve=False, **kwargs):
             The memory_level from which caching must be enabled for the wrapped
             function.
 
-        shelve : bool, default=False
-            Whether to return a joblib MemorizedResult, callable by a .get()
-            method, instead of the return value of func.
-
```

Comment:
`shelve` is still a parameter in method signature. Is there a reason to remove it?